src/main/java/com/minecolonies/colony/buildings/BuildingMiner.java

```diff
@@ -141,14 +148,11 @@ public int getNumberOfLevels()
     /**
      * Returns the current level
      *
-     * @return              Current level
+     * @return Current level
      */
     public Level getCurrentLevel()
     {
-        if(currentLevel >= 0 && currentLevel < levels.size())
-        {
-            return levels.get(currentLevel);
-        }
+        if (currentLevel >= 0 && currentLevel < levels.size()) { return levels.get(currentLevel); }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00122)